samples/Samples.StartupHook.AspNetCoreMvc31/Program.cs

```diff
@@ -0,0 +1,26 @@
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Hosting;
+using Microsoft.Extensions.Logging;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Samples.StartupHook.AspNetCoreMvc31
+{
+    public class Program
```

Comment:
How about adding an integration test that would use this sample to check if `OpenTelemetry.Instrumentation.StartupHook` is working?

Reply:
Great idea! Will get this covered in the next PR.